docs/contributing.md

```diff
@@ -51,11 +51,10 @@ send an email to <security@headlamp.dev> .
 
 Follow these steps when submitting a PR to ensure it meets the project’s standards.
 
-### 1. Run Tests and Format Your Code
+### 1. Run Tests and Formating for your Frontend Code
```

Comment:
This is not required. The word "Format" is the correct spelling (not "Formating"). It's wordier without adding value